tyndale-bulletin.csl

```diff
@@ -284,6 +300,20 @@
       <else-if type="bill book graphic legal_case legislation motion_picture report song" match="any">
         <text variable="title" font-style="italic" text-case="title"/>
       </else-if>
+      <!-- Include the reviewed author for book reviews per §§11.3.8-11.3.9. -->
+      <else-if variable="reviewed-author">
+        <choose>
+          <if variable="reviewed-title">
+            <text variable="title" quotes="true" suffix=", " text-case="title"/>
+            <text variable="reviewed-title" font-style="italic" prefix="review of " suffix=", by " text-case="title"/>
+            <names variable="reviewed-author"/>
```

Comment:
This need a "verb" label I would think

Reply:
I think I have the other comments handled. But my apologies that I'm not following this one. Would you mind clarifying?

In case they're a helpful common point of reference, below are a couple of example bibliography entries that I've been using to test the titled and untitled citation forms.

Akiyama, Kengo. ‘Diverging Teachers of Righteousness’, review of _Sacred Texts and Paradigmatic Revolutions: The Hermeneutical Worlds of the Qumran Sectarian Manuscripts and the Letter to the Romans_, by J. David Stark. _ExpTim_ 126.7 (2015): 358–59.
Das, A. Andrew. Review of _Paul, Judaism, and the Gentiles: Beyond the New Perspective_, by Francis Watson. _Int_ 63 (2009): 200, 202. https://doi.org/10.1177/002096430906300220.

Reply:
Please use the `label` element in `names` to produce "by" rather than a suffix https://docs.citationstyles.org/en/stable/specification.html#label-in-cs-names

Reply:
Thank you. It does make total sense not to hard-code the "by" either.

To misquote the patriarch Abraham, though—let not him who actually knows what's going on be angry with me. But let me ask just once more …

Instead of `label`, would `<text term="by"/>` also be acceptable here with the addition of `<term name="by" form="verb">by</term>` above under `<terms>`? E.g.,

`<text variable="reviewed-title" font-style="italic" suffix="," text-case="title"/>`
`<text term="by"/>`
`<names variable="reviewed-author"/>`

I can get this to work and be valid. But I must be missing something with `names` and `label` since, even when I try just to copy how I'm seeing `label` used inside `names` elsewhere, I end up with a not valid XML error. :-/

Reply:
```suggestion
            <names variable="reviewed-author">
              <label variable="verb" suffix=" "/>
              <name and="text"/>
            </names>  
```

Though I honestly think just using the term is fine in this case (I am wondering, though, if you really don't need a `name` element, e.g. to set and="text". If a book by two authors is reviewed, I assume that's going to be "by Adam Smith and Karl Marx" and not "by Adam Smith, Karl Marx"?